theme/fontsize.css

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2003-2018, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+:root {
+	--ck-fontsize-tiny: .7em;
```

Comment:
Do we need variables for this? We recently decided in some similar feature (maybe even this) to not use variables because overriding styles which we provide boils down to `.ck-content .text-whatever { font-size: whatever }`.

Reply:
See https://github.com/ckeditor/ckeditor5-block-quote/issues/21

Reply:
Probably not but I've made it similar to Highlight feature :stuck_out_tongue:.

I'll fix that.

Reply:
done